### ISSUE TITLE
About Loss function 

### ISSUE THREAD

**Issue by Radio4ctive** [jjjpg/SGANet#1]
Hello mister,
Is not your loss a matrix? I calculated my loss as you have done and then my loss is calculated as 8x8 matrix and some values are inf. When I was trying to loss.backward() it gives an error and it is saying "grad can be implicitly created only for scalar outputs" 
Can you say something about this issue? 
PS: you can send me an email if you want. (otcetinkaya@icloud.com)
Best regards
